packages/ecdh/src/lib.nr

```diff
@@ -0,0 +1,68 @@
+use dep::std::ec::tecurve::affine::{Curve, Point};
+use dep::std::ec::consts::te::{baby_jubjub};
+
+
+// @@@@@@ Core ECDH Implementation
+
+// Globals Edward Curves supported Baby JubJub
+global bjj = baby_jubjub();
+global G = bjj.base8;
```

Comment:
format action check is failing

please put these in a separate file and ignore it from formatting here https://github.com/privacy-scaling-explorations/zk-kit.noir/blob/53815c9fdbb4473622b69ee0fa910bc1a133d861/.dprint.jsonc#L21-L22

see https://github.com/privacy-scaling-explorations/zk-kit.noir/pull/5/files#r1658418842